packages/generator-volto/generators/addon/index.js

```diff
@@ -207,11 +207,6 @@ Run "npm install -g @plone/generator-volto" to update.`,
   }
 
   install() {
-    this.renderTemplate(
-      `${this.templatePath()}/**/*`,
-      this.destinationPath(),
-      this.globals,
-    );
```

Comment:
@instification I think this means that `<%= addonName %>` will no longer be interpolated in Makefile.

Reply:
Thanks! I've refactored to keep the globals so I think this is going to work ok now.